pladder/irc/client.py

```diff
@@ -65,7 +66,8 @@ def update_status(s):
                     else:
                         reply_to = message.sender.nick
                     logger.info("{} -> {} : {}".format(message.sender.nick, target, text))
-                    reply = hooks.on_trigger(message.sender, text[len(config.trigger_prefix):])
+                    text_without_prefix = text[len(config.trigger_prefix):]
+                    reply = hooks.on_trigger(time.time(), config.network, reply_to, message.sender, text_without_prefix)
```

Comment:
time.time() seems to use an unspecified platform-dependent format. Could we use the datetime module? For example, like this: `datetime.now(timezone.utc).timestamp()`

Reply:
Done